src/app/components/marios/marios.component.ts

```diff
@@ -0,0 +1,31 @@
+import { MariosService } from './../../core/services/marios.service';
+import { Component, Input, OnInit } from '@angular/core';
+import { MatDialog } from '@angular/material/dialog';
+import { Marios } from 'src/app/core/interfaces/marios.interface';
+import { MariosModal } from './marios-modal/marios-modal.component';
+
+@Component({
+  selector: 'app-marios',
+  templateUrl: './marios.component.html',
+  styleUrls: ['./marios.component.scss'],
+})
+export class MariosComponent implements OnInit {
+  @Input() marios!: Marios;
+  maxCommentLength: number = 25;
```

Comment:
It is a variable to set maxLength before a "see more" appears. It is an arbitrary value chosen by me, but now I will change the variable name to "maxCommentLengthUntilSeeMore" because the old one was a little bit confusing. Next I have changed the value to 50 characters and allowed the words to break into the next line.